### PR TITLE
Remove: XML support from API Policy Import

### PR DESCRIPTION
**Pull request by BLasan**
## Purpose
- fixes: https://github.com/wso2/api-manager/issues/2069

### CODE REVIEW

**Review by RakhithaRR — APPROVED** [review]

**Comment by BLasan** [comment]
Please do not merge this PR. This needs to be further discussed before merging

**Comment by BLasan** [comment]
Closing this as xml support is available

**Pull request closed without merging**
